<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web服务器管理面板</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>服务器基本设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Web服务器根目录：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="webRootPathLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="webRootPathBrowseBtn">
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>服务器控制：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>8</number>
           </property>
           <item>
            <widget class="QPushButton" name="startServerBtn">
             <property name="text">
              <string>启动服务器</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopServerBtn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>停止服务器</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="serverStatusLabel">
             <property name="text">
              <string>服务器已停止</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>服务器端口：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="setverPortLineEdit">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>8000</string>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>服务器状态</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="logsArea">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
